src/state/atoms/notificationDrawerAtom.ts

```diff
@@ -22,10 +22,13 @@ export type NotificationsPayload = {
   time: string;
 };
 
+export const notificationDrawerScopeReadyAtom = atom(false);
+export const notificationDrawerStateReadyAtom = atom(false);
```

Comment:
I'll make a follow-up PR and remove the old atoms, this one is already pretty big.